### PR TITLE
Point to Gluon example

### PR DESCRIPTION
**Pull request by torkleyy**
Closes https://github.com/slide-rs/specs/issues/520

### CODE REVIEW

**Comment by torkleyy** [comment]
cc @Marwes Are you comfortable having this linked to here? I think the disclaimer that it's incomplete should be enough.

**Comment by Marwes** [comment]
I am fine with linking to it but it should be really clear that it is not something that can be used as-is to get gluon working with specs. It has to much liberal use of unsafe for that so it would be very easy to get undefined behavior with minor changes to it.

Not sure if that fits in with this repo? "Template" sounds misleading to me anyway.

(I have had some ideas since which I need to explore so maybe I will give it another go fairly soon)

File: README.md [hunk 1]
```diff
@@ -26,6 +27,12 @@ Libraries that are generally useful when working with Specs.
 
 ## Specs-powered projects
 
+## Templates
+
+Useful templates that let you get started more easily.
+
+* `shred-example` by @Marwes - (**Incomplete**) game template showing how to use Gluon scripting with Specs [![GitHub](https://img.shields.io/badge/repo-GitHub-green.svg)](https://github.com/Marwes/shred-example) ![incomplete](https://img.shields.io/badge/status-incomplete-blue.svg)
```
Comment (Marwes) [inline README.md:34]: No link here? https://github.com/Marwes/shred-example

Comment (torkleyy) [reply README.md:34]: Yep, it's linked to from the badge

**Comment by torkleyy** [comment]
Thanks for your feedback, maybe it should indeed go into another section.

File: README.md (hunk 1 above)
Comment (Marwes) [reply README.md:34]: Ah, I'm blind.

**Comment by torkleyy** [comment]
Hmm okay, let's close this for now. Made some progress wrt scripting: https://github.com/slide-rs/shred/pull/128

**Pull request closed without merging**